app/src/main/java/tech/ula/utils/ValidationUtility.kt

```diff
@@ -0,0 +1,31 @@
+package tech.ula.utils
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+class ValidationUtility() {
+
+    fun isUsernameValid(password: String): Boolean {
+        val pattern: Pattern
+        val matcher: Matcher
+
+        val PASSWORD_PATTERN = "^[a-zA-Z0-9]*\$"
```

Comment:
This name seems odd to me.

Reply:
Oops it should be renamed to mention `username`

IS is USERNAME_REGEX a better name?

Reply:
`USERNAME_PATTERN` makes the most sense to me, to match the `PASSWORD_PATTERN` below it.